admin_manual/enterprise_user_management/user_auth_shibboleth.rst

```diff
@@ -7,17 +7,45 @@ Introduction
 
 The ownCloud Shibboleth user backend application integrates ownCloud with a 
 Shibboleth Service Provider (SP) and allows operations in federated and 
-single-sign-on infrastructures.
+single-sign-on infrastructures. Setting up Shibboleth has three steps:
+
+1. Create the appropriate Apache configuration
+2. Enable the Shibboleth app
+3. Enable Shibboleth on your ownCloud admin page
 
 Currently supported installations are based on the `native Apache integration`_ 
 . The individual configuration of the service provider is highly dependent on 
 the operating system, as well as on the integration with the Identity 
-Providers (IdP), and deserve case-by-case analysis and installation.
+Providers (IdP), and require case-by-case analysis and installation.
 
 The ownCloud Desktop Client and mobile clients can interact with an 
 ownCloud instance running inside a Shibboleth Service Provider by using built-in 
 browser components for authentication against the IdP.
 
+The ownCloud desktop sync client and mobile apps store users' logins, so 
+your users only need to enter their logins the first time they set up their 
```

Comment:
@danimo ony the first time? AFAIU the browser window will popup whenever the shibboleth token has reached its end of life (lifetime).